URA-859/plots_bcftools.py

```
@@ -0,0 +1,667 @@
+        # Bcftools isec with PASS variants
+        result = bcftools_isec(
+            sp_directory,
+            sample,
+            "allgenesvep_filtered",
+            "old_probes_filtered_PASS",
+            "new_probes_filtered_PASS",
+            "bcftools_isec_PASS",
+        )
+        sp_oldpass_df_uniq = result[0]
+        sp_newpass_df_uniq = result[1]
+        sp_oldpass_df_common = result[2]
```

Ruff (0.8.2)

Local variable `sp_oldpass_df_common` is assigned to but never used

Remove assignment to unused variable `sp_oldpass_df_common`

(F841)